src/AndroidDebugLauncher/AndroidDebugLauncher.csproj

```diff
@@ -48,7 +46,14 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
-  <ItemGroup>
+
+  <ItemGroup Label="Assembly to Generate">
+    <GenerateAssembly Include="$(ILDir)libadb.il">
+      <Visible>false</Visible>
```

Comment:
><Visible>false</Visible> [](start = 6, length = 24)

What winds up consuming `Visible`?

Reply:
Thought this was be for solution explorer. I will double check.